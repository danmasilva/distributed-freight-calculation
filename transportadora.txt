create transportadora t1
update transportadora 0 mudado
