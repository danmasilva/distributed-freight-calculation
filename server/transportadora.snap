create transportadora HelloConcurrent 3 1.4
